src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,7 +5,10 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
-    public Truck() {
+    private String koleso;
+    public Truck(String color , String name , String koleso) {
```

Comment:
The class is missing a no-args constructor. According to the task description, you should not remove the no-args constructor. Please add a no-args constructor to this class.